dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/enrollment/EnrollmentOperationParams.java

```diff
@@ -50,7 +51,7 @@ public class EnrollmentOperationParams {
   @Builder.Default private final EnrollmentParams enrollmentParams = EnrollmentParams.FALSE;
 
   /** Set of te uids to explicitly select. */
-  @Builder.Default private final Set<String> enrollmentUids = new HashSet<>();
+  @Builder.Default private final Set<UID> enrollmentUids = new HashSet<>();
```

Comment:
Now that we use a proper type we should ditch the type suffix from all vars and fields (if its not already taken in the current scope)